examples/tracing/cache_miss.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+#
+# cache_miss.py Example of using BPF perf event.
+#               For Linux, uses BCC, eBPF. Embedded C.
+#
```

Comment:
Need a comment to say something like: this instruments CACHE_MISS, which comes from uapi/linux/perf_event.h, and is really defined for each specific architecture. For example, on x86_64 it means LLC miss (LONGEST_LAT_CACHE.MISS).